conf/test.config

```diff
@@ -22,11 +22,10 @@ params {
     gene_counts = 'https://raw.githubusercontent.com/qbic-pipelines/rnadeseq/dev/testdata/merged_gene_counts.txt'
     model = 'https://raw.githubusercontent.com/qbic-pipelines/rnadeseq/dev/testdata/design.txt'
     genelist = 'https://raw.githubusercontent.com/qbic-pipelines/rnadeseq/dev/testdata/requested_genes.txt'
-    logFCthreshold = 2
+    logFCthreshold = 1
     //report_options = 'https://raw.githubusercontent.com/qbic-pipelines/rnadeseq/dev/testdata/report_options.yml'
```

Comment:
If this is not used, can it be removed?

Reply:
Hm, why would this not be used? It is an active param